Joedmin-hw3/Joedmin-hw3/DuplicateUserCredentialsException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Joedmin_hw3
+{
+    public class DuplicateUserCredentialsException : Exception
+    {
+        public DuplicateUserCredentialsException(string message) : base(message)
```

Comment:
Do you always need to pass a different message in this exception? Maybe you could specify a line or a credential name which is duplicate and build the rest of the message like that?

for example:
```cs
public DuplicateUserCredentialsException(string duplicateUsername) : base($"Username {duplicateUsername} already exists.")
```